watson/cli.py

```diff
@@ -1183,7 +1184,8 @@ def rename(watson, type, old_name, new_name):
         for frame in watson.frames:
             if old_name in frame.tags:
                 watson.frames[frame.id] = frame._replace(
-                    tags=[new_name if t == old_name else t for t in frame.tags]
+                    tags=[new_name if t == old_name else t for t in frame.tags],
```

Comment:
There's a flake8 line-length error here:

Maybe we could make this a bit clearer:

```
                new_tags = [new_name if tag == old_name else tag
                            for tag in frame.tags]
                watson.frames[frame.id] = frame._replace(
                    tags=new_tags,
                    updated_at=updated_at
                )
```

BTW, please make sure that you have no tox failures, including in the flake8 tests, before submitting a PR.